tests/e2e/tests/plugins/PhpPlugin.spec.ts

```diff
@@ -0,0 +1,104 @@
+/*********************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import { WorkspaceNameHandler } from '../..';
+import 'reflect-metadata';
+import * as workspaceHandling from '../../testsLibrary/WorksapceHandlingTests';
+import { e2eContainer } from '../../inversify.config';
+import { CLASSES } from '../../inversify.types';
+import { Ide, LeftToolbarButton } from '../../pageobjects/ide/Ide';
+import { TimeoutConstants } from '../../TimeoutConstants';
+import { TestConstants } from '../../TestConstants';
+import { ProjectTree } from '../../pageobjects/ide/ProjectTree';
+import { Key } from 'selenium-webdriver';
+import { Editor } from '../../pageobjects/ide/Editor';
+import { TopMenu } from '../../pageobjects/ide/TopMenu';
+import { DebugView } from '../../pageobjects/ide/DebugView';
+import { BrowserTabsUtil } from '../../utils/BrowserTabsUtil';
+
+const ide: Ide = e2eContainer.get(CLASSES.Ide);
+const projectTree: ProjectTree = e2eContainer.get(CLASSES.ProjectTree);
+const editor: Editor = e2eContainer.get(CLASSES.Editor);
+const topMenu: TopMenu = e2eContainer.get(CLASSES.TopMenu);
+const debugView: DebugView = e2eContainer.get(CLASSES.DebugView);
+const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
+
+const devfileUrl: string = 'https://raw.githubusercontent.com/eclipse/che/master/tests/e2e/files/devfiles/plugins/PhpPluginTest.yaml';
+const factoryUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/f?url=${devfileUrl}`;
+const projectName: string = 'php-web-simple';
+const subRootFolder: string = 'README.md';
+
+const fileFolderPath: string = `${projectName}`;
+const tabTitle: string = 'index.php';
+
+let workspaceName = 'not defined';
+
+suite(`The 'PhpPlugin' tests`, async () => {
+    suite('Create workspace', async () => {
+        test('Create workspace using factory', async () => {
+            await browserTabsUtil.navigateTo(factoryUrl);
+        });
+
+        test('Wait until created workspace is started', async () => {
+            await ide.waitAndSwitchToIdeFrame();
+            await ide.waitIde(TimeoutConstants.TS_SELENIUM_START_WORKSPACE_TIMEOUT);
+            await projectTree.openProjectTreeContainer();
+            await projectTree.waitProjectImported(projectName, subRootFolder);
+
+            workspaceName = await WorkspaceNameHandler.getNameFromUrl();
```

Comment:
What about moving this directly to the `Delete workspace` test - so the variable initialization and usage would be at one place and there would be no need to make the `workspaceName` variable global for this class. 
E.g. as you did it in another test: https://github.com/eclipse/che/pull/19497/files#diff-6e013c167f0c38d47f74ef27f9a9145ed154e634ae10f5b93c7befca735b6e6aR75